package.json

```diff
@@ -123,6 +123,7 @@
     "es6-promise": "^4.2.8",
     "express": "^4.17.1",
     "express-static-gzip": "^1.1.3",
+    "find-cache-dir": "^3.0.0",
```

Comment:
does this need to be a dependancy? could it be a devDependancy?

Reply:
Nope, i just chucked it in here for now as I couldn't remember what the situation as Re: https://github.com/bbc/simorgh/issues/2531